src/common/telemetry_ops/requests_telemetry.rs

```diff
@@ -29,6 +31,14 @@ pub struct ActixTelemetryCollector {
 #[derive(Default)]
 pub struct ActixWorkerTelemetryCollector {
     methods: HashMap<String, HashMap<HttpStatusCode, Arc<Mutex<OperationDurationsAggregator>>>>,
+    #[allow(clippy::type_complexity)]
+    collections: HashMap<
+        String, /*collection name*/
+        HashMap<
+            String, /*method*/
+            HashMap<HttpStatusCode, Arc<Mutex<OperationDurationsAggregator>>>,
+        >,
+    >,
```

Comment:
Would it be possible to minimize duplicate data here?

Maybe we can omit this in telemetry output, and collect it in a different way when generating /metrics output.